### PR TITLE
Set verbosity if TANZU_CLI_LOG_LEVEL env variable is set

### PR DESCRIPTION
**Pull request by mpanchajanya**
### What this PR does / why we need it

- Logger doesn't pick latest env variable value.
 
### Which issue(s) this PR fixes

Fixes #

### Describe testing done for PR

```

In Tanzu CLI
	log.V(6).Info("testing logger with 6")


mpanchajanya@mpanchajanF09MK ~> tz version
tversion: v1.2.0-dev
buildDate: 2024-02-06
sha: e7e2b8ee-dirty
arch: arm64

# Log is not printed since verbosity is set lower

mpanchajanya@mpanchajanF09MK ~> tz config get
clientOptions:
    features:
        global:
            context-target-v2: "true"
cli:
    ceipOptIn: "true"
    eulaStatus: accepted
    discoverySources:
        - oci:
            name: default
            image: projects.registry.vmware.com/tanzu_cli/plugins/plugin-inventory:latest
    cliId: 736bddac-be88-4084-a00d-137a4972b487
    telemetry:
        source: /Users/mpanchajanya/.config/tanzu-cli-telemetry/cli_metrics.db
mpanchajanya@mpanchajanF09MK ~> tz config set env.TANZU_CLI_LOG_LEVEL 9

# Log is printed if TANZU_CLI_LOG_LEVEL is set since the env variable takes precedence

mpanchajanya@mpanchajanF09MK ~> tz config get
[i] testing logger with 6
clientOptions:
    features:
        global:
            context-target-v2: "true"
    env:
        TANZU_CLI_LOG_LEVEL: "9"
cli:
    ceipOptIn: "true"
    eulaStatus: accepted
    discoverySources:
        - oci:
            name: default
            image: projects.registry.vmware.com/tanzu_cli/plugins/plugin-inventory:latest
    cliId: 736bddac-be88-4084-a00d-137a4972b487
    telemetry:
        source: /Users/mpanchajanya/.config/tanzu-cli-telemetry/cli_metrics.db
mpanchajanya@mpanchajanF09MK ~> tz version
[i] testing logger with 6
version: v1.2.0-dev
buildDate: 2024-02-06
sha: e7e2b8ee-dirty
arch: arm64

# Set log level to lower value

mpanchajanya@mpanchajanF09MK ~> tz config set env.TANZU_CLI_LOG_LEVEL 4
[i] testing logger with 6
mpanchajanya@mpanchajanF09MK ~> tz config get
clientOptions:
    features:
        global:
            context-target-v2: "true"
    env:
        TANZU_CLI_LOG_LEVEL: "4"
cli:
    ceipOptIn: "true"
    eulaStatus: accepted
    discoverySources:
        - oci:
            name: default
            image: projects.registry.vmware.com/tanzu_cli/plugins/plugin-inventory:latest
    cliId: 736bddac-be88-4084-a00d-137a4972b487
    telemetry:
        source: /Users/mpanchajanya/.config/tanzu-cli-telemetry/cli_metrics.db
mpanchajanya@mpanchajanF09MK ~>


```

### Release note

```release-note
 Logger verbosity can be changed by setting TANZU_CLI_LOG_LEVEL environment variable.
```

### CODE REVIEW

File: pkg/command/root.go [hunk 1]
```diff
@@ -9,10 +9,12 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"strconv"
 	"strings"
 
 	"github.com/google/uuid"
 	"github.com/pkg/errors"
+
```
Comment (anujc25) [inline pkg/command/root.go:17]: nit: extra line not needed here.

**Review by anujc25 — APPROVED** [review]
LGTM. Thanks for the quick fix. (minor nit, feel free to ignore)

File: pkg/command/root.go [hunk 2]
```diff
@@ -210,6 +215,18 @@ func newRootCmd() *cobra.Command {
 	return rootCmd
 }
 
+// SetLoggerVerbosity sets the verbosity of the logger if TANZU_CLI_LOG_LEVEL is set
+func SetLoggerVerbosity() {
```
Comment (marckhouzam) [inline pkg/command/root.go:219]: No need to make this public, right? `SetLoggerVerbosity` -> `setLoggerVerbosity`

**Review by marckhouzam — APPROVED** [review]
Small nit also but LGTM anyway

File: pkg/command/root.go (hunk 2 above)
Comment (mpanchajanya) [reply pkg/command/root.go]: changed to private method

**Pull request merged**